src/Common/Uuid/UuidMapping.php

```diff
@@ -18,9 +18,19 @@
 
 use OpenEMR\Common\Database\QueryUtils;
 use OpenEMR\Common\Uuid\UuidRegistry;
+use OpenEMR\Services\FHIR\Observation\FhirObservationVitalsService;
+use OpenEMR\Services\FHIR\Observation\FhirObservationSocialHistoryService;
 
 class UuidMapping
 {
+    private const UUID_MAPPING_DEFINITIONS = [
+        ['resource' => 'CareTeam', 'table' => 'patient_data'],
+        ['resource' => 'Location', 'table' => 'patient_data'],
+        ['resource' => 'Location', 'table' => 'users'],
+        ['resource' => 'Location', 'table' => 'facility'],
+        ['resource' => 'Observation', 'table' => 'form_vitals', 'codes' => FhirObservationVitalsService::COLUMN_MAPPINGS, 'category' => FhirObservationVitalsService::CATEGORY],
```

Comment:
The work I'm doing may remove the need for the form_vitals table mapping, FYI, I'll know more today if that's going to be possible.